### PR TITLE
feat: config node email service

### PR DESCRIPTION
**Pull request by cowan-macady**
# Checklist

- [ ] `make test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by cowan-macady** [comment]
pushed by part to limit number of lines to review at a same time

File: jarvis_sdk/model/email_service_config.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        )
+
+        if message_config.HasField('sendgrid'):
+            email_service_config.sendgrid = SendGridProviderConfig(api_key=message_config.sendgrid.api_key,
+                                                                   sandbox_mode=message_config.sendgrid.sandbox_mode,
+                                                                   ip_pool_name=message_config.sendgrid.ip_pool_name,
+                                                                   host=message_config.sendgrid.host)
+
+        if message_config.HasField('authentication_message'):
+            email_service_config.authentication_message = EmailDefinition(template=EmailTemplate.deserialize(message_config.authentication_message.template),
```
Comment (martasd) [inline jarvis_sdk/model/email_service_config.py:26]: Looks like this line is way too long. It's good practice to respect the maximum line length, which should be set in each project.

File: jarvis_sdk/cmdconfig/_config_node.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    return response.config_node
+
+
+def update_email_service_config_node(self, config_node_id, etag, display_name, description, email_service_config, bookmarks=[]):
+
+    try:
+        response = self.stub.UpdateConfigNode(
+            pb2.UpdateConfigNodeRequest(
+                id=config_node_id, etag=wrappers.StringValue(value=etag), display_name=wrappers.StringValue(value=display_name),
```
Comment (martasd) [inline jarvis_sdk/cmdconfig/_config_node.py:55]: It would be more readable if each argument was on a separate line.

File: .coveragerc [hunk 3]
```diff
@@ -9,6 +9,17 @@ omit =
...
+  jarvis_sdk/model/config_node.py
+  jarvis_sdk/model/create_config_node.py
+  jarvis_sdk/model/email.py
+  jarvis_sdk/model/email_attachment.py
+  jarvis_sdk/model/email_message.py
+  jarvis_sdk/model/email_service_config.py
+  jarvis_sdk/model/email_template.py
+  jarvis_sdk/model/sendgrid_email_provider.py
+  jarvis_sdk/model/update_config_node.py
+
```
Comment (martasd) [inline .coveragerc:22]: Why should all of these be omitted?

File: jarvis_sdk/model/email_service_config.py (hunk 1 above)
Comment (cowan-macady) [reply jarvis_sdk/model/email_service_config.py:26]: ok

File: jarvis_sdk/cmdconfig/_config_node.py (hunk 2 above)
Comment (cowan-macady) [reply jarvis_sdk/cmdconfig/_config_node.py:55]: ok

**Pull request closed without merging**